docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+      "text/plain": [
+       "<Figure size 1444.95x200.667 with 1 Axes>"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Decomposition of an RZZ(θ) gate using the IBM Quantum QPU basis\n",
+    "service = QiskitRuntimeService()\n",
+    "backend = service.least_busy()\n",
+    "optimization_level = 3\n",
+    "pm = generate_preset_pass_manager(optimization_level, backend=backend)\n",
+    "transpiled_circuit = pm.run(qc)\n",
+    "transpiled_circuit.draw('mpl', idle_wires=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "22007518-b6fb-4341-b483-894feb5d8947",
+   "metadata": {},
+   "source": [
+    "For workflows that require many single qubit $R_X(\\theta)$ or two-qubit rotations, especially ones where the rotation angle $\\theta$ is small (such as in a variational ansatze or when simulating the time evolution of quantum systems), this constraint causes the circuit depth to grow quite quickly. Thankfully however, the fleet of IBM Quantum QPUs now support *fractional gates* which can significantly reduce the circuit depth for workflows like these."
```

Comment:
I think this info could be brought out in the first paragraph before launching into the code examples, otherwise I feel like I have to read a lot before I get to the point. I think generally a better format for short-attention-span-readers (i.e. most docs readers lets be honest 😆) is to make the point in the first paragraph and then evidence it with code examples second